### ISSUE TITLE
[Bug]: Apps' translated descriptions are not used in App Store

### ISSUE THREAD

**Issue by Jerome-Herbinet** [nextcloud/server#38813]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Apps' translated descriptions (from Transifex) in App Store are not used.

I don't know if the problem's source is the same as in the public app store (web, outside Nextcloud instances) : https://github.com/nextcloud/appstore/issues/691

IMO, a particular attention should be paid to this problem for marketing reasons. When a description appears translated, it's more reassuring, more salesy. 

I'm not specifying a version of Nextcloud, because if I remember correctly, the problem has always been there, or almost. At least seen in NC 25 26 27 ...

### Steps to reproduce

1.
2.
3.

### Expected behavior

Translated descriptions should appear.

### Installation method

None

### Nextcloud Server version

27

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

**Comment by Jerome-Herbinet** [comment]
@szaimen do you have any idea who could take care of this ?

**Comment by szaimen** [comment]
No not really. The appstore was gladly rakwn over by @bigcat88. So cc @Pytal and @JuliaKirschenheuter for the settings app.

**Comment by bigcat88** [comment]
yeah, but not so fast, we still in process deciding how will look the deployment process, only after that all fixes will start arriving

**Comment by Jerome-Herbinet** [comment]
> yeah, but not so fast, we still in process deciding how will look the deployment process, only after that all fixes will start arriving

Hello @bigcat88 ; any news ?

**Comment by oleksandr-nc** [comment]
Unfortunately, there is no news, we have not made any progress on this.

To push translations from Transifex for applications in the AppStore is a completely new feature, and not a small one.

The main maintainer of the AppStore is @edward-ly now, - I am interested to know his opinion(when he has time) on this issue and in which ways we can implement such option.

**Comment by edward-ly** [comment]
Yes, I agree that grabbing the translations from Transifex would not be easy. Not to mention, there are some community apps that use different translation providers altogether (some examples mentioned here: https://github.com/nextcloud/appstore/issues/691#issuecomment-621765083), so the solution should ideally be translation provider-agnostic.

Actually, the preferred way to add translations would be to just include them all in the `appinfo/info.xml` file, like so:

```
<name>News</name>
<name lang="de">Nachrichten</name>
<name lang="ja">ニュース</name>
```

Repeat as desired for the other languages you want to include and for the `<summary>` and `<description>` tags. This is detailed in the [app store documentation](https://nextcloudappstore.readthedocs.io/en/latest/developer.html#app-metadata). If the translations still don't show up after that, then please report that as it's clearly a bug. If you believe an alternative solution is better, let us know why you think that should be the case.
